dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java

```diff
@@ -729,4 +706,44 @@ private void handleDragDetectedEvent(final Event event) {
         event.consume();
     }
 
+    private void startPutJob(final Session session,
+            final List<Pair<String, Path>> files,
+            final String bucket,
+            final String targetDir,
+            final JobInterruptionStore jobInterruptionStore) {
+        final Ds3Client client = session.getClient();
```

Comment:
Since this call only uses the client from the session, why don't we pass in the client instead of the session?